mongorestore/mongorestore_archive_test.go

```diff
@@ -132,3 +138,174 @@ func TestMongorestoreBadFormatArchive(t *testing.T) {
 		})
 	})
 }
+
+// ----------------------------------------------------------------------
+// All tests from this point onwards use testify, not convey. See the
+// CONTRIBUING.md file in the top level of the repo for details on how to
+// write tests using testify.
+// ----------------------------------------------------------------------
+
+func TestMongorestoreArchiveAdminNamespaces(t *testing.T) {
+	require := require.New(t)
+
+	testtype.SkipUnlessTestType(t, testtype.IntegrationTestType)
+
+	session, err := testutil.GetBareSession()
+	require.NoError(err, "can connect to server")
+
+	fcv := testutil.GetFCV(session)
+	if cmp, err := testutil.CompareFCV(fcv, "4.2"); err != nil || cmp < 0 {
```

Comment:
Looks like these tests aren't compatible with the MMAPv1 storage engine- I think setting this to 4.2 is OK because atlas shared clusters only run 6.0?

Reply:
Yeah I think that's fine. What was the error you saw though?

Reply:
```
error demultiplexing archive; archive finished but contained files were unfinished ([admin.system.indexes])
```

Reply:
Hmmm I'm not sure how that's specific to MMAPv1. Can you link me to the failing tests on evergreen?

Reply:
Here's an [example](https://spruce.mongodb.com/task/mongo_tools_amazon_integration_3.4_mmapv1_patch_369ced42d4aad50c1036534f3a541051f7618863_64b0513c57e85aa71dcf0dc9_23_07_13_19_32_13/tests?execution=0&sortBy=STATUS&sortDir=ASC)